.github/workflows/publish.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish Python package
+# Releases are generated in the release.yml workflow
+
+on:
+  release:
+    types: [published]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
```

Comment:
I guess this could go in the other CI and the results uploaded as artifacts or something, but the idea still stands (building for different platforms).

Reply:
you’re right, we should use one of the examples instead: https://github.com/PyO3/maturin-action?tab=readme-ov-file#examples

Reply:
This is kind of what I meant on standup - the maturin one works with all of this stuff for free even if it is kind of weird it doesn't use UV.  That's what I was saying about peeling back just the `pip` stuff and inserting `uv`. You should be able to get it with a `maturin generate-ci github --pytest > .github/workflows/CI.yml`.